sematic/plugins/abstract_kuberay_wrapper.py

```diff
@@ -84,6 +91,36 @@ def _get_ray_version() -> str:
         )
 
 
+@dataclass(frozen=True)
+class AutoscalerConfig:
+    """Configuration for the autoscaler.
+
+    Note that the autoscaler will execute in the same pod with the head node,
+    if the autoscaler is enabled.
+
+    Attributes
+    ----------
+    cpu:
+        Number of CPUs for each node (supports fractional CPUs).
+    memory_gb:
+        Gigabytes of memory for each node (supports fractional values).
```

Comment:
```suggestion
    cpu: float
        Number of CPUs for each node (supports fractional CPUs).
    memory_gb: float
        Gigabytes of memory for each node (supports fractional values).
```

Reply:
That's (a) inconsistent with other examples in this file (b) not necessary since the types are already documented via type hints.